nautobot/extras/templates/extras/configcontextschema.html

```diff
@@ -0,0 +1,105 @@
+{% extends 'base.html' %}
+{% load buttons %}
+{% load custom_links %}
+{% load helpers %}
+{% load plugins %}
+
+{% block header %}
+    <div class="row noprint">
+        <div class="col-sm-8 col-md-9">
+            <ol class="breadcrumb">
+                <li><a href="{% url 'extras:configcontextschema_list' %}">Config Context Schemas</a></li>
+                <li>{{ object }}</li>
+            </ol>
+        </div>
+        <div class="col-sm-4 col-md-3">
+            <form action="{% url 'extras:configcontextschema_list' %}" method="get">
+                <div class="input-group">
+                    <input type="text" name="q" class="form-control" />
+                    <span class="input-group-btn">
+                        <button type="submit" class="btn btn-primary">
+                            <span class="mdi mdi-magnify" aria-hidden="true"></span>
+                        </button>
+                    </span>
+                </div>
+            </form>
+        </div>
```

Comment:
I don't think I'm familiar with this bit of boilerplate - what does this provide?

Reply:
This just adds the search box on the detail view that takes you back to the list view for that model. That said, the input is actually missing the placeholder that should say "Search config context schemas" in this case.